wdl/ExpansionHunter.wdl

```diff
@@ -87,13 +93,22 @@ task RunExpansionHunter {
     command <<<
         set -euxo pipefail
 
+        sex=""
+        if [ ! -z "~{ped_filename}" -a "~{ped_filename}" != " " ]; then
+            if [ -e "~{ped_filename}" ]; then
+                sex=$(awk -F '\t' '{if ($2 == "~{individual_id}") {if ($5 == "1") {print "--sex male"; exit 0} else if ($5 == "2") {print "--sex female"; exit 0} else {exit 1}}}' < ~{ped_filename} )
```

Comment:
I would actually try to emit a helpful error message if the sex is undefined. I think it would be best to define `sex="NA"` if the ped file is provided. Then use `print "NA"; exit 0` instead of `exit 1` in the last `if` clause. Then check the value of `sex` below - if it's `NA` then warn that either the sample id was not found or the sex is undefined and fail.

Reply:
Please check if the updated condition would be equally good.